### PR TITLE
fix(content-explorer): remove inline-editing in metadata view

### PR DESCRIPTION
**Pull request by manishPh**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @manishPh has signed the CLA. Thanks for the pull request!

File: src/features/metadata-based-view/MetadataBasedItemList.js [hunk 1]
```diff
@@ -285,7 +159,13 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         });
 
         return (
-            <div key={key} className={classes} style={style}>
+            <div
+                key={key}
+                className={classes}
+                style={style}
+                onMouseLeave={this.handleMouseLeave}
+                onMouseEnter={() => this.handleMouseEnter(columnIndex, rowIndex)}
```
Comment (sahiga) [inline src/features/metadata-based-view/MetadataBasedItemList.js]: Minor: can we alphabetize these attributes?

File: src/features/metadata-based-view/MetadataBasedItemList.js [hunk 2]
```diff
@@ -133,61 +93,18 @@ class MetadataBasedItemList extends React.Component<Props, State> {
...
     }
 
-    handleEditIconClick(columnIndex: number, rowIndex: number, value: string): void {
-        this.setState({
-            editedColumnIndex: columnIndex,
-            editedRowIndex: rowIndex,
-            valueBeingEdited: value,
-        });
-    }
+    handleMouseEnter = (columnIndex: number, rowIndex: number): void => this.setState({ hoveredRowIndex: rowIndex });
```
Comment (sahiga) [inline src/features/metadata-based-view/MetadataBasedItemList.js]: Do we still need `columnIndex` here?

File: src/features/metadata-based-view/MetadataBasedItemList.js (hunk 1 above)
Comment (manishPh) [reply src/features/metadata-based-view/MetadataBasedItemList.js]: @sahiga done!

File: src/features/metadata-based-view/MetadataBasedItemList.js (hunk 2 above)
Comment (manishPh) [reply src/features/metadata-based-view/MetadataBasedItemList.js]: @sahiga done!

**Pull request merged**